alembic/versions/4cb9e98344b2_gym_and_user_activity_update.py

```diff
@@ -0,0 +1,47 @@
+"""gym and user_activity update
+
+Revision ID: 4cb9e98344b2
+Revises: ff8bb0abd100
+Create Date: 2016-05-16 12:51:39.519120
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '4cb9e98344b2'
+down_revision = 'ff8bb0abd100'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+from sqlalchemy.dialects import postgresql
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('gym',
+                    sa.Column(
+                        'id', sqlalchemy_utils.types.uuid.UUIDType(), nullable=False),
+                    sa.Column('name', sa.String(length=100), nullable=True),
+                    sa.Column('city', sa.String(length=100), nullable=True),
+                    sa.PrimaryKeyConstraint('id')
+                    )
+    op.add_column('user_activity', sa.Column(
+        'gym_id', sqlalchemy_utils.types.uuid.UUIDType(), nullable=True))
+    op.add_column('user_activity', sa.Column(
+        'raining_outside', sa.Boolean(), nullable=True))
```

Comment:
deze staat nu in Weather right? Dus hoeft niet meer aangemaakt te worden in deze